ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/PartStack.java

```diff
@@ -145,6 +146,9 @@
    */
   void setDelegate(ActionDelegate delegate);
 
+  /** Returns related to Part Stack properties */
+  PartStackProperties getProperties();
```

Comment:
Also would be more simple return here just a Map<String, String>

Reply:
I see your point, thank you, Vitalii!
We can have some issue related to manage and control properties at this case. Also we should notify consumers  when some property is changed.
I have changed implementation so we can avoid extra objects.
Please review.